src/main/java/nextstep/courses/domain/SessionEnrolment.java

```diff
@@ -5,26 +5,31 @@
 public class SessionEnrolment {
 
     private SessionStudent sessionStudent;
-    private SessionStatusType sessionStatusType;
+    private SessionStatus sessionStatus;
     private Amount amount;
     private boolean isFree;
 
     public SessionEnrolment(SessionStudent sessionStudent, SessionStatusType sessionStatusType, Amount amount, boolean isFree) {
+        this(sessionStudent, new SessionStatus(sessionStatusType), amount, isFree);
+    }
+
+    public SessionEnrolment(SessionStudent sessionStudent, SessionStatus sessionStatus, Amount amount, boolean isFree) {
         this.sessionStudent = sessionStudent;
-        this.sessionStatusType = sessionStatusType;
+        this.sessionStatus = sessionStatus;
         this.amount = amount;
         this.isFree = isFree;
     }
 
-    public void freeEnrolment(NsUser student) {
-        defaultValidate();
-        this.sessionStudent.add(student);
-    }
-
-    public void payEnrolment(NsUser student, Long userPayed) {
+    public void enrolment(Long userPayed) {
+        if (isFree) {
+            defaultValidate();
+            return;
+        }
         defaultValidate();
         validatePay(userPayed);
+    }
 
+    public void addStudent(NsUser student) {
         this.sessionStudent.add(student);
```

Comment:
```suggestion
        this.sessionStudents.add(student);
```

복수의 의미가 없는데 add하는게 개인적으론 어색하네요 😂

Reply:
그렇군요... 복수형으로 클래스명을 수정했습니다! https://github.com/next-step/java-lms/pull/293/commits/0326758017623f4441842709f97330b47f6186c7